src/allmydata/client.py

```diff
@@ -818,6 +824,10 @@ def init_storage(self, announceable_storage_servers):
         if anonymous_storage_enabled(self.config):
             furl_file = self.config.get_private_path("storage.furl").encode(get_filesystem_encoding())
             furl = self.tub.registerReference(FoolscapStorageServer(ss), furlFile=furl_file)
+            (_, _, swissnum) = furl.rpartition("/")
```

Comment:
Sometime real soon we should probably have a real library for parsing furls and nurls.

Reply:
https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3932

Reply:
(Although there is _some_ fURL parsing, at least, so I will switch to using that).